scripts.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+window.addEventListener('load', main);
+document.addEventListener('click', countClick);
```

Comment:
if you use main pattern, will be better to add the listener to document inside main